sofl/onlinefix_game.py

```
@@ -77,304 +69,158 @@ def launch(self) -> None:
+
+    def _handle_uninstall_response(self, response: str, game_root: Path) -> None:
+        """Обрабатывает ответ пользователя в диалоге удаления"""
+        if response != "uninstall":
+            return
+
+        self.log_and_toast(_("{} started uninstalling").format(self.name))
+
+        try:
+            shutil.rmtree(game_root)
+            self.log_and_toast(_("{} uninstalled").format(self.name))
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)